File: app/src/main/java/org/ole/planet/myplanet/callback/OnLibraryItemSelected.java
package org.ole.planet.myplanet.callback;

import org.ole.planet.myplanet.model.RealmMyLibrary;
import org.ole.planet.myplanet.model.RealmTag;

import java.util.List;

public interface OnLibraryItemSelected {
    void onSelectedListChange(List<RealmMyLibrary> list);
    void onTagClicked( RealmTag realmTag);
}

File: app/src/main/java/org/ole/planet/myplanet/callback/OnLibraryItemSelected.kt
package org.ole.planet.myplanet.callback

import org.ole.planet.myplanet.model.RealmTag
import org.ole.planet.myplanet.model.dto.LibraryItem

interface OnLibraryItemSelected {
    @JvmSuppressWildcards
    fun onSelectedListChange(list: MutableList<LibraryItem>)
    @JvmSuppressWildcards
    fun onTagClicked(realmTag: RealmTag)
    @JvmSuppressWildcards
    fun onItemClicked(libraryId: String)
}
